hasura/migrations/default/1704993161810_create_table_pending_app_metadata/up.sql

```diff
@@ -1,26 +1,28 @@
-CREATE TABLE "public"."pending_app_metadata" (
-  "id" varchar NOT NULL DEFAULT gen_random_friendly_id('update'),
+CREATE TABLE "public"."app_metadata" (
+  "id" varchar NOT NULL UNIQUE DEFAULT gen_random_friendly_id('update'),
+  "app_id" varchar NOT NULL,
   "name" varchar NOT NULL,
-  "logo_img_url" text NOT NULL,
-  "showcase_img_urls" jsonb NOT NULL,
-  "hero_image_url" text NOT NULL,
+  "logo_img_url" varchar NULL,
+  "showcase_img_urls" text[] NULL,
+  "hero_image_url" varchar NULL,
   "description" text NOT NULL,
-  "category" varchar NOT NULL,
-  "publisher" varchar NOT NULL,
-  "link_to_integration" varchar NOT NULL,
-  "review_message" text NOT NULL,
-  "review_status" varchar NOT NULL,
-  "is_developer_allow_listing" bool NOT NULL,
-  "world_app_description" varchar NOT NULL,
-  "team_id" varchar NOT NULL,
+  "world_app_description" varchar NULL,
```

Comment:
generally not a good idea to allow NULL in varchar fields because you then have two values representing empty/nil, empty string should be enough. important to make sure the default is an empty string